DEV-Simple/core/MediaManager.swift

```diff
@@ -67,6 +67,32 @@ class MediaManager: NSObject {
         avPlayer?.seek(to: CMTime(seconds: seconds, preferredTimescale: CMTimeScale(NSEC_PER_SEC)))
     }
 
+    private func seekForward(_ sender: Any) {
+        guard let duration  = avPlayer?.currentItem?.duration else {
+            return
+        }
+        let playerCurrentTime = CMTimeGetSeconds(avPlayer!.currentTime())
```

Comment:
`avPlayer!` feels a bit insecure to me even though I'm confident it should always be instantiated by this point. So not that I consider it necessary, but how do you feel about declaring `playerCurrentTime` in a guard clause to be even safer regarding this?